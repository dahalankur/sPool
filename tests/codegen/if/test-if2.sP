if (true && true && (false && (true || false || true))):
    print("nope")
else
    print("Hey there!")
    print(" My name Jeff")
    println(".")
;