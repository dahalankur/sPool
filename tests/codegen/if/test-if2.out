Hey there! My name Jeff.
